cuenca_validations/types/queries.py

```diff
@@ -84,10 +84,11 @@ class StatementQuery(QueryParams):
 
     @validator('year', 'month')
```

Comment:
You only need to put one or the other. That'd also remove the need for the `if` statement